dist/test_data.py

```diff
@@ -254,5 +254,10 @@ def __ge__(self, other):
     'search_near_02' : Method(test_config),
     'search_near_03' : Method(test_config),
     'test_template' : Method(test_config),
+    'cursor_bound_04' : Method(test_config),
```

Comment:
Would be awesome if we could automate that, but can we order A-Z please?

Reply:
[ae9841c](https://github.com/wiredtiger/wiredtiger/pull/8185/commits/ae9841c0fa914a2701c2f7db422fcdbc87f696e6)